funfact/lang/_terminal.py

```diff
@@ -176,8 +179,10 @@ def vectorize(self, n, append):
         else:
             initializer = self.initializer[..., None] if append else \
                           self.initializer[None, ...]
+        prefer = vmap(self.prefer, append) if self.prefer else None
```

Comment:
Based on our discussion today, we can explore later whether it is possible to unify `stack` and `vmap`.

Reply:
Makes sense, I'll create a separate issue for this.